HIRS_Utils/src/main/java/hirs/tpm/TPMBaselineGenerator.java

```diff
@@ -641,6 +637,10 @@ private void parseBaseline(final String baselineName, final InputStream inStream
                 TPMBaselineFields.toOSInfo(fieldMap, baseline.getOSInfo()));
             baseline.setTPMInfo(
                 TPMBaselineFields.toTPMInfo(fieldMap, baseline.getTPMInfo()));
+
+            if (baseline.isEmpty()) {
+                throw new Exception("TPM baseline is empty!");
```

Comment:
I would make this a `TpmBaselineGeneratorException` for consistency with the rest of the exception in this method, but I'm curious, were we allowed to have empty rows in CSVs before? Does this check preclude that from happening?

Like for example every baseline in the CSV is separated by an empty row.

Reply:
The isEmpty() method was written to address the case where a csv file with valid data terminates with an empty line.  Previously, this case did not throw an error because the while loop truncated the last line of each file, empty or not.  With the modified while loop both an empty last line and a blank csv throw an error.  isEmpty() differentiates the two cases.